librz/core/agraph.c

```diff
@@ -3762,6 +3757,18 @@ RZ_API RzANode *rz_agraph_add_node_with_color(const RzAGraph *g, const char *tit
 	return res;
 }
 
+/**
+ * \brief Convert a RzGraphNodeInfo \p info to RzANode and add to \p g.
+ */
+RZ_API RzANode *rz_agraph_add_node_from_node_info(RZ_NONNULL const RzAGraph *g, RZ_NONNULL const RzGraphNodeInfo *info) {
```

Comment:
If you add `RZ_NONNULL`, please add also `rz_return_val_if_fail`, please also ownership mark on the return value.